src/components/PDFViewer/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO: get from site branding
+import settings from '../../data/configuration/constants';
+
+import './PDFViewer.scss';
+
+const getPDFUrl = relativeUrl => (
+  `${settings.journalsBackendBaseUrl}${relativeUrl}`
+);
+
+const getIFrameUrl = relativeUrl => (
+  `${settings.journalsBackendBaseUrl}/static/pdf_js/web/viewer.html?file=${getPDFUrl(relativeUrl)}`
```

Comment:
Anything in an iframe exists in the scope of that domain (as if you were on that webpage directly), so you don't need to prepend the domain to the relative URL, you can just use the relative URL directly.

Reply:
ah i didn't realize that, thanks for letting me know - i'll fix that up